packages/backend/src/controllers/message.ts

```diff
@@ -8,9 +8,12 @@ import {
 } from '@util/response'
 import { Request, Response } from 'express'
 
-export const getAllMessagesController = async (_: Request, res: Response) => {
+export const getAllMessagesController = async (req: Request, res: Response) => {
   try {
-    const messages: MessageDoc[] = await getAllMessages()
+    // "/messages?lastId=[lastId]&limit=[limit]"
```

Comment:
you can remove this comment and instead include it in the description of this PR

Reply:
We'll include that information in API documentation later

Reply:
Ok, I'll do that ina second